pipelines/skylab/multiome/Multiome.wdl

```diff
@@ -18,7 +18,7 @@ workflow Multiome {
         File annotations_gtf
         File ref_genome_fasta
         File? mt_genes
-        Int tenx_chemistry_version = 3
+        Int tenx_chemistry_version = 2
```

Comment:
@nikellepetrillo Why was the multiome changed to 2? I thought that Multiome chemistry was v3?

Reply:
Thanks for catching this. Changing it back ( i think my local copy of warp is messed up, im going to have to delete and clone again :( )